pyports/cluster_activities_utils.py

```diff
@@ -81,7 +112,7 @@ def polygenize_clusters_with_features(type_of_area_mapped, df_for_clustering, po
     for cluster in tqdm(df_for_clustering.cluster_label.unique()):  # iterate over clusters
         record = {}
         cluster_df = df_for_clustering[df_for_clustering.cluster_label == cluster]  # sub-df for chosen cluster
-        points = cluster_df[[f'{blip}Blip_lng', f'{blip}Blip_lat']].to_numpy()  # numpy array of lng/lat
+        points = cluster_df[[f'{blip}Blip_lon', f'{blip}Blip_lat']].to_numpy()  # numpy array of lon/lat
 
         if optimize_polygon:
```

Comment:
This feature flag is good for research. We would like to have the best results - is it with or without the polygon optimisation? If we decide to go with both options they both should appear one after the other